package.json

```diff
@@ -56,6 +56,7 @@
   },
   "scripts": {
     "prepare": "husky install && (forge --version || yarn foundryup) && yarn workspaces run prepare",
+    "build": "yarn workspaces run prepare",
```

Comment:
feels a bit odd to have the global `build` script mapped to packages' `prepare` script, should we map it to their `build` script instead?

Reply:
oh yep!

this was a bit of a temporary workaround because (afaict) there's no way to tell yarn to "run script in all workspaces, only if its defined" and I thought `prepare` was what we wanted but turns out everything just runs `yarn build` inside their `prepare`, so will change!